README.md

```diff
@@ -32,39 +32,49 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+Document Object Model (DOM) allows you to manipulate html and css through javascript.
```

Comment:
That's part of it, but what is it really? Check this: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction